robottelo/ui/activationkey.py

```diff
@@ -175,6 +175,27 @@ def add_host_collection(self, name, host_collection_name):
             raise UINoSuchElementError(
                 "Couldn't find activation key '{0}'".format(name))
 
+    def remove_host_collection(self, name, host_collection_name):
+        """Remove an existing Host Collection from Activation Key."""
+        # find activation key
+        activation_key = self.search(name)
```

Comment:
And again copy-paste from method above. Thanks for code sample.
Perhaps it is required to revise all such helpers as separate issue?